cmd/ndt7-client/main.go

```diff
@@ -115,9 +115,9 @@ const (
 )
 
 var (
-	ClientName	= "ndt7-client-go-cmd"
-	ClientVersion	= "0.6.1"
-	flagProfile = flag.String("profile", "",
+	ClientName    = "ndt7-client-go-cmd"
+	ClientVersion = "0.6.1"
```

Comment:
I think it may be worth bumping the version number (maybe 0.7.0), document and make a release

Reply:
I agree -- the version bump will be a separate PR once this one is merged.